src/shared/sources/nyt/index.js

```diff
@@ -58,8 +58,8 @@ module.exports = {
             } else {
               locationObj.county = row.county //geography.getCounty(row.county, row.state)
 
-              if (locationObj.county === '(unassigned)') {
```

Comment:
To be clear, the string `(unassigned)` never appears in NYT's dataset. As such, this branch would never be taken. The string value we should check for here is `Unknown`.